driver/main.go

```diff
@@ -9,7 +9,9 @@ import (
 
 func main() {
 	driver.Run(driver.Transforms{
-		Native: normalizer.Native,
-		Code:   normalizer.Code,
+		Preprocess: normalizer.Preprocess,
```

Comment:
I don't see the same change for tests. Might be the reason why `.sem.uast` are the same as `.uast` ;)

Reply:
Yes, that was it! Probably could be useful for the skeleton to define the `driver.Transforms` in a separate var and use it both from `main` and `fixtures` (the user could always change the one in fixtures if needed).

Reply:
Regarding the Transform, I'll add it to SDK. Forgot about this issue.